src/test/ui-fulldeps/plugin-args-2.rs

```diff
@@ -1,10 +1,8 @@
-// run-pass
-// aux-build:plugin-args.rs
+// check-pass
```

Comment:
File can be deleted; it's identical to `plugin-args-1.rs`.

Reply:
Well, there are extra argument parentheses in `plugin-args-2`.
I'll remove it together with support for plugin arguments.